middlewares/errors/unauthorized.js

```diff
@@ -14,7 +14,7 @@ const unauthorized = (err, req, res, next) => {
   }
 
   return res.status(UNAUTHORIZED).json({
-    ok: false,
+    success: false,
```

Comment:
❓ Is the `ok` that was removed used anywhere on FE?

Reply:
Those messages are not used anywhere on FE. I changed them for consistency. These functions might not even be used on BE, that could be part of a cleanup refactoring. Added it to BE refactor trello card.